RNSentryAndroidTester/.gitignore

```diff
@@ -0,0 +1,15 @@
+*.iml
```

Comment:
M: We already have `RNSentryTester`, so either we rename `RNSentryTester` to `RNSentryCocoaTester` or we create a folder for both testers so the naming can be more specific.